components/pipeline-service/staging/base/main-pipeline-service-configuration.yaml

```diff
@@ -1562,6 +1562,18 @@ spec:
                       requests:
                         cpu: 200m
                         memory: 200Mi
+        tekton-pipelines-controller:
+          spec:
+            template:
+              spec:
+                containers:
+                  - name: tekton-pipelines-controller
+                    resources:
+                      requests:
+                        memory: 500Mi
```

Comment:
You have to set this to the value you want and run the generate script to update the deploy.yaml files. It looks like you've edited them manually and the next time someone runs the generate script, it'll override those values.